bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestWorkItem.java

```diff
@@ -59,7 +59,7 @@ abstract class PullRequestWorkItem implements WorkItem {
         this.bot = bot;
         this.prId = prId;
         this.errorHandler = errorHandler;
-        this.prUpdatedAt = prUpdatedAt;
+        this.triggerUpdatedAt = prUpdatedAt;
```

Comment:
I like the new naming of this field, but please also update the parameter in the constructor.

Reply:
Sure